omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -91,3 +91,10 @@
 </form>
 
 <div id="patient-search-results"></div>
+
+<% if (showRegisterPatientLink == "true") { %>
+<div>
+${ ui.message("coreapps.findPatient.registerPatient.label") }&nbsp;&nbsp;<a id="patient-search-register-patient" class="button" href="${ui.pageLink("registrationapp", "registerPatient", ["appId":"referenceapplication.registrationapp.registerPatient"])}">${ui.message("registrationapp.registration.label")}</a>
```

Comment:
This is problematic; you have a reference to a referenceapplication app, but the coreapps module may also be used without the reference application. (E.g. PIH does this.)

Didn't we have a discussion about this, and the good-enough approach is to introduce a global property for "what is the main register patient app"?